jme3-desktop/src/main/java/com/jme3/system/NativeLibraryLoader.java

```diff
@@ -125,23 +124,6 @@ public static void registerNativeLibrary(String name, Platform platform,
     }
     
     static {
-        // LWJGL
-        registerNativeLibrary("lwjgl", Platform.Windows32, "native/windows/lwjgl.dll");
-        registerNativeLibrary("lwjgl", Platform.Windows64, "native/windows/lwjgl64.dll");
-        registerNativeLibrary("lwjgl", Platform.Linux32,   "native/linux/liblwjgl.so");
-        registerNativeLibrary("lwjgl", Platform.Linux64,   "native/linux/liblwjgl64.so");
-        registerNativeLibrary("lwjgl", Platform.MacOSX32,  "native/macosx/liblwjgl.dylib");
-        registerNativeLibrary("lwjgl", Platform.MacOSX64,  "native/macosx/liblwjgl.dylib");
-        
-        // OpenAL
-        // For OSX: Need to add lib prefix when extracting
-        registerNativeLibrary("openal", Platform.Windows32, "native/windows/OpenAL32.dll");
-        registerNativeLibrary("openal", Platform.Windows64, "native/windows/OpenAL64.dll");
-        registerNativeLibrary("openal", Platform.Linux32,   "native/linux/libopenal.so");
-        registerNativeLibrary("openal", Platform.Linux64,   "native/linux/libopenal64.so");
-        registerNativeLibrary("openal", Platform.MacOSX32,  "native/macosx/openal.dylib", "libopenal.dylib");
-        registerNativeLibrary("openal", Platform.MacOSX64,  "native/macosx/openal.dylib", "libopenal.dylib");
-        

```

Comment:
Removing this can cause issues. The SDK is relying on being able to extract natives even though the context is not initialized. Also I strongly suggest using a different library name instead of `"lwjgl"` for lwjgl3, probably should just use `"lwjgl3"` to avoid conflicts. For openal you can use `"lwjgl3-openal"` instead. Once LWJGL 3.0 becomes official / stable we can then get rid of LWJGL 2 and replace the lwjgl and openal libraries with their LWJGL 3.0 equivalents.